djangocms_page_meta/models.py

```diff
@@ -86,6 +88,12 @@ def copy_relations(self, oldinstance, language):
             item.page = self
             item.save()
 
+    @property
+    def robots_list(self):
+        if self.robots:
+            return ast.literal_eval(self.robots)
```

Comment:
do we have a better way than `ast` to do this?

doesn't `self.robots.split(",")` achieve the same goal?

Reply:
@yakky It is a bit more complicated than this, because Django stores that value as a string like `"['foo', 'bar']"` . Anyway, I'll try to remove `ast` and come with a different solution

Reply:
@yakky This is the equivalent of the `ast` : `return self.robots[1:-1].replace("'", "").split(", ")` . If we like this more, I'll change that.

Reply:
doesn't `json.loads(self.robots)` works?

Reply:
Will try and  report back

Reply:
@yakky This will do the trick `json.loads(self.robots.replace("'", "\""))` ... is this ok?